src/sentry/static/sentry/app/components/splitLayout.jsx

```diff
@@ -4,16 +4,25 @@ import classNames from 'classnames';
 import SpreadLayout from './spreadLayout';
 
 // Flexbox, use when you want your children to be equal sizes
-const SplitLayout = ({children, className, responsive, ...props}) => {
-  const cx = classNames('split-layout', className, {
+const SplitLayout = ({children, className, responsive, splitWidth, ...props}) => {
+  let cx = classNames('split-layout', className, {
     'allow-responsive': responsive
   });
+  let childCount = 0;
+  let totalChildren = React.Children.count(children);
```

Comment:
do you know why react has count, foreach and map instead of just toArray? is there some special reason for maintaining shouldupdate mechanics or something??

Reply:
Not entirely sure, I assume they need to do additional processing when they traverse children.